actors/common/src/main/java/org/sunbird/common/quartz/scheduler/OnDemandSchedulerManager.java

```diff
@@ -0,0 +1,84 @@
+package org.sunbird.common.quartz.scheduler;
+
+import org.quartz.*;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+
+public class OnDemandSchedulerManager extends SchedulerManager{
+
+  private static OnDemandSchedulerManager onDemandSchedulerManager = null;
+
+  public static Map<String, String> schedulerMap = new HashMap<>();
+
+  private OnDemandSchedulerManager() {
+   super();
+   if(schedulerMap == null) {
+    schedulerMap.put("bulkupload","uploadVerify");
+    schedulerMap.put("shadowuser","migrateShadowUser");
+   }
+  }
+
+  private void scheduleOnDemand(String identifier, String job) {
+   ProjectLogger.log(
+           "onDemandSchedulerManager:scheduleOnDemand:scheduler started",
```

Comment:
@Hari-stackroute can we print job name in the logs, easy to debug

Reply:
ok, done.